### DISCUSSION TITLE
Why are Editor.js plugins installed as dev dependencies?

### DISCUSSION THREAD

**Discussion by LeonGeldsch** [codex-team/editor.js#2479] [Q&A]
In the documentation of some editor.js plugins, it says to install it as a dev dependency. For me this doesn't make sense, since it is not something just used in development and will be included in the final bundle and later sent to the clients. Also for some plugins like the [header plugin](https://github.com/editor-js/header) it says to install it as a normal dependency while for others, like the [image plugin](https://github.com/editor-js/image), it says to install it as a dev dependency.

**Comment by btrewern** [comment]
I think this is just bad documentation.  I have always installed them as normal dependencies and have not had any issues.

**Reply by Dorfeuheinz** [reply]
could you elaborate the process? im finding it difficult to understand whats dist & wheres the bundle.js file located? Id be hihgly obliged
